### ISSUE TITLE
[Bug] Movies wont play on Flatpak Version

### ISSUE THREAD

**Issue by Super-JoConnell** [CorsixTH/CorsixTH#2361]
### Describe the issue

The flatpak version of Corsix-TH 0.66 has a bug where the movies won't play either it be the intro movies or the level movies. the terminal output doesn't log anything relating to the movies unless debug option is enabled then i get the following terminal output:

> MoviePlayer:playMovie - Warning: Invalid data found when processing input
> Cannot check for updates since LuaSocket and/or LuaSec are not available.
> starting game with difficulty full
> Created game with savegame version 170.
> MoviePlayer:playMovie - Warning: Invalid data found when processing input

### Steps to Reproduce

1. Install Flatpak

2. Enable Movies In Settings

3. Run Software as normal Movies fail to play

### Expected Behaviour

Movies to play when enabled 

### Save Game

N/A

### CorsixTH Version

0.66 Flatpak

### Operating System

SteamOS

### Theme Hospital Version

GOG.com

### Gamelog.txt

```bash
None created, couldn't find gamelog.txt at flatpak location
```

### Additional Information

![Screenshot_20230625_103940](https://github.com/CorsixTH/CorsixTH/assets/49617291/ffd81cd1-0d5b-4e36-95b0-a2ea287f27c8)

**Comment by lewri** [comment]
@colinjmatt @rmnvgr did we confirm this the other week?

**Comment by colinjmatt** [comment]
This seems to be an issue with the flatpak itself. It's very likely that the bundled ffmpeg libraries need looking at.

The issue with being unable to check for updates will likely be due to luasec not being present.

**Comment by Alberth289346** [comment]
If there was really something wrong with movies in Corsix itself in 0.66, we would have many bug reports about it, given the age of that version. As that is not the case, a more likely candidate is that something went wrong with flatpak bundling.
The best people to ask about it are the people that produce the flatpak.

It is of course possible that they have hit a bug in Corsix in some way. Also then, they are likely the people best equipped to investigate and/or report about that bug.

**Comment by lewri** [comment]
If bug is upstream we may wish to transfer this issue to flathub/com.corsixth.corsixth

**Comment by Super-JoConnell** [comment]
sorry was unsure who to contact about this issue, didn't realise there was a github for the flatpak, thanks for opening the ticket across there, do you wish for me to close the issue here ?

**Comment by lewri** [comment]
Issue has been identified so the flatpak is hopefully resolved by 0.67 stable (soon

**Comment by lewri** [comment]
Resolved by  flathub/com.corsixth.corsixth#25
